packages/algob/src/lib/files.ts

```diff
@@ -64,3 +65,10 @@ export function loadFromYamlFileSilentWithMessage (
     return defaultYamlValue(options);
   }
 }
+
+export function writeTransaction (transactionConfirmation: Object): void {
+  fs.appendFileSync(
+    path.join(artifactsPath, "transactionInfo.txt"),
+    YAML.stringify(transactionConfirmation)
+  );
+}
```

Comment:
let's remove this function. We need more things and this is scheduled for other task: https://www.pivotaltracker.com/story/show/174085665

FYI: we will write the output of all transactions in a file with filename related to the script filename. 
So we need more abstraction here (probably a `TxLogWriter` class, which will  be part of the deployer and can be used by a user as well.
Here let's only do printing to a console

Reply:
ok